Examples/Algorithms/TrackFinding/src/TrackFindingAlgorithmTrackFinderFunction.cpp

```diff
@@ -69,10 +69,10 @@ ActsExamples::TrackFindingAlgorithm::makeTrackFinderFunction(
         navigator.resolveMaterial = true;
         navigator.resolveSensitive = true;
         Propagator propagator(std::move(stepper), std::move(navigator));
-        CKF trackFinder(std::move(propagator));
+        Ckf trackFinder(std::move(propagator));
```

Comment:
Cosmetic change, and arbuably, worse. Would you also do `using Kf = KalmanFilter`?

Reply:
Fixed.